reactos/dll/win32/browseui/shellbrowser.cpp

```diff
@@ -1147,42 +1216,62 @@ HRESULT CShellBrowser::ShowBand(const CLSID &classID, bool vertical)
     CComPtr<IDeskBar>                       deskBar;
     VARIANT                                 vaIn;
     HRESULT                                 hResult;
+    DWORD                                   dwBandID;
 
     hResult = GetBaseBar(vertical, (IUnknown **)&theBaseBar);
     if (FAILED_UNEXPECTEDLY(hResult))
         return hResult;
-
-#if USE_CUSTOM_EXPLORERBAND
-    TRACE("ShowBand called for CLSID %s, vertical=%d...\n", wine_dbgstr_guid(&classID), vertical);
-    if (IsEqualCLSID(CLSID_ExplorerBand, classID))
-    {
-        TRACE("CLSID_ExplorerBand requested, building internal band.\n");
-        hResult = CExplorerBand_Constructor(IID_PPV_ARG(IUnknown, &newBand));
-        if (FAILED_UNEXPECTEDLY(hResult))
-            return hResult;
-    }
-    else
-#endif
-    {
-        TRACE("A different CLSID requested, using CoCreateInstance.\n");
-        hResult = CoCreateInstance(classID, NULL, CLSCTX_INPROC_SERVER, IID_PPV_ARG(IUnknown, &newBand));
-        if (FAILED_UNEXPECTEDLY(hResult))
-            return hResult;
-    }
+    
+    // Try to grab IDeskbar's interface of our basebar
     hResult = theBaseBar->QueryInterface(IID_PPV_ARG(IDeskBar, &deskBar));
     if (FAILED_UNEXPECTEDLY(hResult))
         return hResult;
+
+    // Get the deskBar's client
```

Comment:
It was a hint for myself while trying to figure out the flow of basebar/basebarsite creation (since I didn't know nothing about that before). But yes, it needs to be removed now :)